src/semafour.cc

```diff
@@ -14,9 +14,14 @@ using v8::Persistent;
 using v8::String;
 using v8::Uint32;
 using v8::Value;
+using v8::False;
+using v8::True;
 
 Persistent<Function> Semafour::constructor;
 
+#define TRYWAIT_OK     0
```

Comment:
Can you make these `const int`, and name them `kTryWaitOk` and `kTryWaitLocked`.